modules/item/index.php

```diff
@@ -101,17 +103,19 @@
 		}
 
 		if ($equipLoc !== false && $equipLoc !== '-1') {
-			if(is_numeric($equipLoc) && (floatval($equipLoc) == intval($equipLoc))) {
-				$equipLocationCombinations = Flux::config('EquipLocationCombinations')->toArray();
-				if (array_key_exists($equipLoc, $equipLocationCombinations) && $equipLocationCombinations[$equipLoc]) {
+			$equipLocs = explode('/', $equipLoc);
+
+			if($equipLoc && count($equipLocs) == 1) {
+				$equipLocations = Flux::config('EquipLocations')->toArray();
+				if (array_key_exists($equipLoc, $equipLocations) && $equipLocations[$equipLoc]) {
 					if ($equipLoc === '0') {
-						$sqlpartial .= "AND (equip_locations = 0 OR equip_locations IS NULL) ";
+						$sqlpartial .= "AND ($equipLoc = 0 OR $equipLoc IS NULL) ";
 					} else {
-						$sqlpartial .= "AND equip_locations = ? ";
-						$bind[]      = $equipLoc;
+						$sqlpartial .= "AND $equipLoc = ? ";
+						$bind[]      = 1;
 					}
 				}
-			} else {
+			}/* else {
 				$combinationName = preg_quote($equipLoc, '/');
 				$equipLocationCombinations = preg_grep("/.*?$combinationName.*?/i", Flux::config('EquipLocationCombinations')->toArray());
 				
```

Comment:
Should we remove it or what?

Reply:
combination search does not work now.
it needs to be rewritten or deleted.